sources/isal/igzip/igzip.h

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ ******************************************************************************/
+
+#ifndef QPL_SOURCES_ISAL_IGZIP_IGZIP_C
+#define QPL_SOURCES_ISAL_IGZIP_IGZIP_C
+
+// includes extracted from sources/isal/igzip/igzip.c
+#include <assert.h>
+#include <string.h>
+#ifdef _WIN32
+# include <intrin.h>
```

Comment:
These are still present in `igzip.c`, so the change doesn't really make sense to me. Also, not all declarations/externs were moved to the header, so really I'd just prefer 3 declarations to be inserted into the `igzip.c` directly.